### DISCUSSION TITLE
cannot add extra info from Query

### DISCUSSION THREAD

**Discussion by rainbow2019s** [fastapi/fastapi#8756] [Questions]
WTForm can add helper info custom desc,but I can not  find in Query,Path and Body

**Comment by Mause** [comment]
What extra info are you referring to?

**Comment by rainbow2019s** [comment]
@app.get('/api') #/api?limit=100
async def admin(
    limit:int=Query(...,ge=1,le=10,description='参数错误')
):
    return {'limit':limit}

@app.exception_handler(RequestValidationError)
async def validation_exception_handler(request, exc):
    # print(type(exc.errors()))
    return JSONResponse(
        status_code=400,
        content={
            'code':'10001',
            'message':exc.errors()[0]['msg'], 
            'data':{}}
    )

I want to get description in exception_handler to replace default msg
